Ports/ntfs-3g/package.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='ntfs-3g'
+version='2022.5.17'
+useconfigure='true'
+files="https://github.com/tuxera/ntfs-3g/archive/refs/tags/${version}.zip ntfs-3g-${version}.zip d953594f4d947ebe0ddf34740926e5508f440b868b5381fa6e51e491824fb5ba"
```

Comment:
Actual release tarballs are hosted at `https://tuxera.com/opensource/ntfs-3g-${version}.tgz`, those also don't appear to require autotools.